common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -1,42 +1,22 @@
 {
-    "env": {
-        "browser": true,
-        "es2024": false,
-        "es6": true,
-        "node": true
-    },
-    "globals": {
-        "Atomics": "readonly",
-        "SharedArrayBuffer": "readonly"
-    },
-    "ignorePatterns": [
-        "**/packageVersion.ts",
-        "**/layerGenerationState.ts",
-        "**/*.generated.ts",
-        "**/*.generated.js"
-    ],
-    "parserOptions": {
-        "ecmaFeatures": {
-            "jsx": true
-        },
-        "ecmaVersion": "latest",
-        "project": "./tsconfig.json",
-        "sourceType": "module"
+    "language": "@/js",
+    "linterOptions": {
+        "reportUnusedDisableDirectives": 1
     },
     "plugins": [
+        "@",
```

Comment:
Is this line correct?

Reply:
ESLint 9's flat config registers its built-in rules as a plugin named `@`